### PR TITLE
Update guidelines for creating bug tickets

### PR DESCRIPTION
**Pull request by amyblais**
- Update guidelines for how to open bug tickets
 - Add guideline to fill in Severity for all bug tickets

Related to [Reactive Tickets Analysis & Optimization](https://docs.google.com/document/d/1aPIfVVvsNAY01PCleeUSy7N2ZP_rY279tJYC7FpaDrk/edit#bookmark=id.glzylg42e769) project.

### CODE REVIEW

**Review by cwarnermm — DISMISSED** [review]

File: operations/research-and-development/product/development-process/new-bug-tickets/bug-severity-guidelines.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
 ## Criteria for bugs
 
 * “Scale” of impact
-  * S1 - `Data loss or crash` - Prevents app use (e.g. Mattermost crashes when posting a message). These are the only bugs that can be committed after the T-5 Code Freeze deadline.
-  * S2 - `Functionality loss or glaring UI regressions` - Most users’ functionality affected (e.g. cannot favorite a channel) or includes functional and UI regressions that affect functionality. In addition to S1 bugs, these are the only bugs that can be committed after Code Complete deadline until T-5 Code Freeze.
```
Comment (marianunez) [inline operations/research-and-development/product/development-process/new-bug-tickets/bug-severity-guidelines.md:7]: Are these bugs not longer accepted as cherry pick after code complete deadline?

**Review by lindalumitchell — DISMISSED** [review]
Thanks Amy! Looks good to me. I'm curious about Maria's question as well, but I don't have any suggestions for changes to the text. 👍

**Review by wiersgallak — DISMISSED** [review]
LGTM

File: operations/research-and-development/product/development-process/new-bug-tickets/bug-severity-guidelines.md (hunk 1 above)
Comment (amyblais) [reply operations/research-and-development/product/development-process/new-bug-tickets/bug-severity-guidelines.md:7]: I've added back that sentence as it makes sense to keep it here.

**Review by esarafianou — APPROVED** [review]
LGTM

**Review by marianunez — APPROVED** [review]

**Pull request merged**